torch/_inductor/lowering.py

```diff
@@ -5737,7 +5745,7 @@ def create_placeholder(name: str, dtype: torch.dtype) -> InputBuffer:
             ]:
                 sdpa_template.maybe_append_choice(
                     choices=choices,
-                    input_nodes=(query, key, value),
+                    input_nodes=(query, key, value, logsumexp),
```

Comment:
btw, we need some way for the template to say which one of the inputs are being mutated.